src/mod/applications/mod_conference/conference_video.c

```diff
@@ -4834,13 +4834,15 @@ void conference_video_write_frame(conference_obj_t *conference, conference_membe
 {
 	conference_member_t *imember;
 	int want_refresh = 0;
-	unsigned char buf[SWITCH_RTP_MAX_BUF_LEN] = "";
```

Comment:
can we keep this on stack but make it the right size instead.  This gets called a LOT.